app/code/Magento/Newsletter/Controller/Manage/Save.php

```diff
@@ -110,7 +111,7 @@ public function execute()
                 $this->messageManager->addError(__('Something went wrong while saving your subscription.'));
             }
         }
-        $this->_redirect('customer/account/');
+        return $this->_redirect('customer/account/');
```

Comment:
Why this is needed?

Reply:
I think this is logical due to lib/internal/Magento/Framework/App/ActionInterface.php:30